README.md

```diff
@@ -50,6 +50,7 @@ A RAG orchestration framework.
 
 Check out the Ragna documentation to get started:
 
+- [Overview video](https://youtu.be/xuNnjFS1XqQ?si=AssQq_1CF2o1jv3M)
```

Comment:
GitHub Flavored Markdown does not support `<iframe>` to embed the video, so adding a link instead. :)